docs/src/userguide/memoryviews.rst

```diff
@@ -691,4 +691,3 @@ call functions in C files, see :ref:`using_c_libraries`.
 .. _new style buffers: http://docs.python.org/c-api/buffer.html
 .. _pep 3118: http://www.python.org/peps/pep-3118.html
 .. _NumPy: http://docs.scipy.org/doc/numpy/reference/arrays.ndarray.html#memory-layout
-.. _example: http://www.scipy.org/Numpy_Example_List#newaxis
```

Comment:
I consider this page a good replacement: https://docs.scipy.org/doc/numpy/reference/arrays.indexing.html

Reply:
Thanks for the link! I think we should still keep both the example and the link.